RealmSwift/Tests/TestCase.swift

```diff
@@ -33,6 +33,7 @@ class TestCase: XCTestCase {
         Realm.defaultPath = realmPathForFile("\(realmFilePrefix()).default.realm")
         NSFileManager.defaultManager().createDirectoryAtPath(realmPathForFile(""), withIntermediateDirectories: true, attributes: nil, error: nil)
 
+        internalImplementation().setNumberOfTestIterations(1)
```

Comment:
That actually isn't possible in Swift 1.1, since you can't ever call through to `super` inside a closure (which has been fixed in 1.2)